lang/perl5/Portfile

```diff
@@ -55,6 +55,11 @@ foreach {perl5.v perl5.subversion perl5.revision perl5.rmd160 perl5.sha256 perl5
         depends_lib-append  port:db48 \
                             port:gdbm
 
+        # https://trac.macports.org/ticket/66478
+        if {${perl5.major} < 5.28} {
+            platforms           {darwin < 22} freebsd linux
```

Comment:
It might make sense to put this with the related code around line 214/219. Up to Mojca though.

Reply:
done